tracer/src/Datadog.Trace.MSBuild/DatadogLogger.cs

```diff
@@ -102,7 +103,7 @@ private void EventSource_BuildStarted(object sender, BuildStartedEventArgs e)
 
                 _buildSpan = _tracer.StartSpan(BuildTags.BuildOperationName);
                 _buildSpan.SetMetric(Tags.Analytics, 1.0d);
-                _buildSpan.SetTraceSamplingPriority(SamplingPriorityValues.AutoKeep);
+                _buildSpan.SetTraceSamplingDecision(SamplingPriorityValues.AutoKeep, SamplingMechanism.CiApp);
```

Comment:
@tonyredondo: What was the goal here by setting this? To force all the traces to be sampled?